File: php/Clothing.php
<?php

class Clothing {
    private $size;
    private $material;
    private $gender;

    // Parameterized constructor
    public function __construct($size = "", $material = "", $gender = "") {
        $this->size = $size;
        $this->material = $material;
        $this->gender = $gender;
    }

    // Getters
    public function getSize() {
        return $this->size;
    }

    public function getMaterial() {
        return $this->material;
    }

    public function getGender() {
        return $this->gender;
    }

    // Setters
    public function setSize($size) {
        $this->size = $size;
    }

    public function setMaterial($material) {
        $this->material = $material;
    }

    public function setGender($gender) {
        $this->gender = $gender;
    }
}

?>

File: python/Clothing.py
class Clothing:
    def __init__(self, size="", material="", gender=""):
        self.__size = size
        self.__material = material
        self.__gender = gender

    def get_size(self):
        return self.__size

    def set_size(self, size):
        self.__size = size

    def get_material(self):
        return self.__material

    def set_material(self, material):
        self.__material = material

    def get_gender(self):
        return self.__gender

    def set_gender(self, gender):
        self.__gender = gender